build.gradle.kts

```diff
@@ -18,8 +18,8 @@ import org.jetbrains.compose.desktop.application.dsl.TargetFormat
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-    kotlin("jvm") version "1.5.10"
-    id("org.jetbrains.compose") version "0.5.0-build227"
+    kotlin("jvm") version "1.5.31"
```

Comment:
Why not upgrade to the latest Kotlin release (`1.6.10`) is it not compatible with compose?